tests/test_outputs.py

```diff
@@ -975,6 +975,19 @@ def test_motor_pins_nonpwm(mock_factory):
         assert motor.backward_device.pin is b
         assert isinstance(motor.backward_device, DigitalOutputDevice)
 
+def test_motor_pins_enable_pwm(mock_factory, pwm):
+    s = mock_factory.pin(1)
+    f = mock_factory.pin(2)
+    b = mock_factory.pin(3)
+    with Motor(f, b, enable=s, pwm=False, enable_is_pwm=True) as motor:
```

Comment:
Looking at the other Motor tests here, it seems that the Motor constructor is passed explicit numbers rather than pin objects? i.e. should this be
```python3
    with Motor(2, 3, enable=1, pwm=False, enable_is_pwm=True) as motor:
```

(and obviously the same for the other tests you've added too)